### PR TITLE
added Ci cd pipeline process 

### PR DESCRIPTION
**Pull request by RamanaChowdam**
added docs for release management and ci cd process

### CODE REVIEW

File: learn/app-development/deployment/configure-pipelines.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: "Configure Pipeline (WME only)"
+id: ""
+---
+---
+
+wavemaker support three phases pipeline for WME customers, are QA,Stage and Live.user can configure each phase with different profile configuratons and with other settings.with one click deployment application is deploy to the QA phase by default.Before pushing the application to the next phase user has to configure the phases in Launchapad at App Deployment section.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/deployment/configure-pipelines.md:7]: needs lot changes

File: learn/app-development/deployment/configure-pipelines.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+For WME customer need to configure pipeline before deploying application.visit [pipeline configuraton](../../on-premise/configure/config-pipeline.md) for App Deployment.
+
+With one click deployment application is deploy to the QA phase by default.Before pushing the application to the next phase user has to configure the phases in Launchapad at App Deployment section for deploying into the Demo wavemaker cloud and wavemaker also support major cloud provicers AWS,AZURE ,GCP and Digital ocean kubernetes services.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/deployment/configure-pipelines.md:11]: needs lots of changes

**Comment by prasanth-reddy-wavemaker-com** [comment]
@swethakundaram let me know when i should review this

**Comment by swethakundaram** [comment]
@prasanth-reddy-wavemaker-com , will do.

**Comment by rnaresh** [comment]
**Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology - 20 Such Instances**

2020-08-31T05:17:36.108Z [INFO]: /codebuild/output/src684543564/src/docs/learn/app-development/deployment/default-pipelines.md
                                 7:1   ✓ error  Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology
                                 8:32  ✓ error  Incorrect usage of the term: “Wavemaker”, use “WaveMaker” instead  terminology
                                 /codebuild/output/src684543564/src/docs/learn/app-development/deployment/deployment-using-docker-compose.md
                                 9:1    ✓ error  Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology
                                 11:67   ✓ error  Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology
                                 51:86   ✓ error  Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology
                                 51:123  ✓ error  Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology
                                 51:203  ✓ error  Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology
                                 89:32   ✓ error  Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology
                                 /codebuild/output/src684543564/src/docs/learn/app-development/deployment/export-wavemaker-application.md
                                 9:16  ✓ error  Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology
                                 /codebuild/output/src684543564/src/docs/learn/app-development/deployment/integration-with-custom-vcs-provider.md
                                 20:122  ✓ error  Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology
                                 /codebuild/output/src684543564/src/docs/learn/app-development/deployment/jenkins-integration.md
                                 7:1    ✓ error  Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology
                                 7:108  ✓ error  Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology
                                 18:200  ✓ error  Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology
                                 /codebuild/output/src684543564/src/docs/learn/app-development/deployment/phase-configuration.md
                                 7:1  ✓ error  Incorrect usage of the term: “Wavemaker”, use “WaveMaker” instead  terminology
                                 /codebuild/output/src684543564/src/docs/learn/app-development/deployment/push-code-to-external-repo.md
                                 7:1  ✓ error  Incorrect usage of the term: “Wavemaker”, use “WaveMaker” instead  terminology
                                 11:1  ✓ error  Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology
                                 /codebuild/output/src684543564/src/docs/learn/app-development/deployment/release-management.md
                                 45:203  ✓ error  Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology
                                 /codebuild/output/src684543564/src/docs/learn/app-development/deployment/tests-integration.md
                                 11:113  ✓ error  Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology
                                 /codebuild/output/src684543564/src/docs/learn/app-development/deployment/webhooks-integration.md
                                 11:64   ✓ error  Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology
                                 11:111  ✓ error  Incorrect usage of the term: “wavemaker”, use “WaveMaker” instead  terminology

**Review by swethakundaram — APPROVED** [review]

**Comment by swethakundaram** [comment]
> @swethakundaram let me know when i should review this

Can you review it now?

**Comment by swethakundaram** [comment]
> @swethakundaram let me know when i should review this

@prasanth-reddy-wavemaker-com you can review it now.

**Comment by swethakundaram** [comment]
@rnaresh the PR failed because it says "the build was canceled" do you know why?

**Comment by swethakundaram** [comment]
@jvenugopal @prasanth-reddy-wavemaker-com @krishnaraoch
You can access to the latest content for the whole deployment flow from here: https://pr-304.d2ulx3keu6fdjg.amplifyapp.com/learn/app-development/deployment/deployment-overview

Once approved, we merge it.

**Review by prasanth-reddy-wavemaker-com — APPROVED** [review]

**Review by prasanth-reddy-wavemaker-com — APPROVED** [review]

**Pull request merged**